content/en/specifications/_index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Specification
+linkTitle: Specification
+menu: { main: { weight: 35 } }
+cascade:
+  type: docs
+---
+
+The TUF spec minimizes key compromise impact by supporting roles with multiple
+keys and threshold trust, while ensuring that highly vulnerable online keys are
+not used for trusted roles. Designed for flexibility, it is easy to integrate
+with various software update systems.
```

Comment:
I'm not sure that I can make sense of this paragraph. I'd suggest replacing it quite simply by the list of specification versions:

```suggestion
The following specification versions are available:

- [latest](https://theupdateframework.github.io/specification/latest/index.html)
- [draft](https://theupdateframework.github.io/specification/draft/index.html)
- [v1.0.33](https://theupdateframework.github.io/specification/v1.0.33/index.html)
- [v1.0.32](https://theupdateframework.github.io/specification/v1.0.32/index.html)
- [v1.0.31](https://theupdateframework.github.io/specification/v1.0.31/index.html)
- [v1.0.30](https://theupdateframework.github.io/specification/v1.0.30/index.html)
- [v1.0.29](https://theupdateframework.github.io/specification/v1.0.29/index.html)
- [v1.0.28](https://theupdateframework.github.io/specification/v1.0.28/index.html)
- [v1.0.27](https://theupdateframework.github.io/specification/v1.0.27/index.html)
- [v1.0.26](https://theupdateframework.github.io/specification/v1.0.26/index.html)
- [v1.0.25](https://theupdateframework.github.io/specification/v1.0.25/index.html)
- [v1.0.24](https://theupdateframework.github.io/specification/v1.0.24/index.html)
- [v1.0.23](https://theupdateframework.github.io/specification/v1.0.23/index.html)
- [v1.0.22](https://theupdateframework.github.io/specification/v1.0.22/index.html)
- [v1.0.20](https://theupdateframework.github.io/specification/v1.0.20/index.html)
- [v1.0.19](https://theupdateframework.github.io/specification/v1.0.19/index.html)
- [v1.0.18](https://theupdateframework.github.io/specification/v1.0.18/index.html)
- [v1.0.17](https://theupdateframework.github.io/specification/v1.0.17/index.html)

This list is also available from
[Spec versions](https://theupdateframework.github.io/specification/).
```